python/tools/delocate-wheel

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+"""Monkey patching delocate wheel support exclude.
+
+See
+https://github.com/matthew-brett/delocate/blob/master/delocate/cmd/delocate_wheel.py
+"""
+
+import os
+import sys
+from argparse import ArgumentParser
+
+from delocate import delocate_wheel
+
+
+def lib_filter_func(name):
+    ignored_paths = ["/usr/lib", "/opt/homebrew", "/usr/local"]
+    for path in ignored_paths:
+        if name.startswith(path):
+            return False
+    return True
+
+
+def main():
+    parser = ArgumentParser()
+    parser.add_argument("--require-archs", metavar="ARCH")
+    parser.add_argument("-w", "--wheel-dir")
+    parser.add_argument("wheel", nargs="+")
+
+    args = parser.parse_args()
+    print(args)
+    for wheel in args.wheel:
+        out_dir = args.wheel_dir
```

Comment:
What happens if out_dir is None? Should this be required?

Reply:
fixed.